tensorboard/plugins/debugger/debugger_plugin_loader.py

```diff
@@ -91,42 +125,46 @@ def load(self, context):
     Returns:
       A DebuggerPlugin instance or None if it couldn't be loaded.
     """
-    if not (context.flags.debugger_data_server_grpc_port > 0 or
-            context.flags.debugger_port > 0):
-      return None
     flags = context.flags
-    try:
-      # pylint: disable=g-import-not-at-top,unused-import
-      import tensorflow
-    except ImportError:
-      raise ImportError(
-          'To use the debugger plugin, you need to have TensorFlow installed:\n'
-          '  pip install tensorflow')
-    try:
+    if flags.debugger_data_server_grpc_port > 0 or flags.debugger_port > 0:
+      # Verify that the required Python packages are installed.
+      try:
+        # pylint: disable=g-import-not-at-top,unused-import
+        import tensorflow
+      except ImportError:
+        raise ImportError(
+            'To use the debugger plugin, you need to have TensorFlow installed:\n'
+            '  pip install tensorflow')
+
+    if flags.debugger_data_server_grpc_port > 0:
```

Comment:
Question about the behavioral change: it seems like debugger_data_server_grpc_port takes precedence over debugger_port and noninteractive_plugin is chosen when both are passed. Is that desired in terms of UX? Also, did we explicitly mention that in the README or flags help? I don't think it is that obvious just reading the flag help.

Reply:
The help message for the flag `--debugger_data_server_grpc_port` says it's mutually exclusive with `--debugger_port`:
https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/debugger/debugger_plugin_loader.py#L53

The logic that checks the mutual exclusion is at:
https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/debugger/debugger_plugin_loader.py#L80